src/Hedgehog/ListGen.fs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Hedgehog
+
+module ListGen =
+
+    let traverse (f: 'a -> Gen<'b>) (ma: list<'a>) : Gen<list<'b>> =
```

Comment:
```suggestion
    let traverse (f : 'a -> Gen<'b>) (ma : list<'a>) : Gen<list<'b>> =
```

Reply:
The F# style guide says to only put white space on the right side of those colons.
https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#right-pad-function-argument-type-annotations

Reply:
Be that as it may, the rest of the code in this code base doesn't do that. I value internal consistency over edicts :smile:

Reply:
We could probably investigate #118 in a follow up PR to address style concerns like this, it would definitely be beneficial.

Reply:
I don't mind the formatting change.  I just wanted to provide the link for reference.

Reply:
I have my style preferences, but I most concerned with consistency.

Reply:
> I don't mind the formatting change. I just wanted to provide the link for reference.

Understood.

> I have my style preferences, but I most concerned with consistency.

Same. I wouldn't write code the way it's written in this code base, but I seldom break with tradition on matters like this.

Reply:
Most of the style is back from 2016-2017 by the way. Is there some official style/formatting guide by Microsoft or by the F# community?

Reply:
Yes.  [Microsoft has an F# style guide](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/).

Reply:
I vote we apply the style guide as a separate PR so we can consider it on its own merits without having to also consider logic changes. I'm in favor of using the style guide, but I don't want to block features.

Reply:
(FWIW, item no. 3 from that style guide originates from https://twitter.com/dsymetweets/status/843849405212098560.)